### ISSUE TITLE
printer loss with 4.6.2

### ISSUE THREAD

**Issue by PolyEsterFhil** [bleachbit/bleachbit#1728]
Curious to know (known issues) if there is an area in 4.6.2 which causes printer issues.  I've had them before but have been able to get them resolved, but with the newest version I've been unable to get it back up and running.  Any help is deeply appreciated!!!

**Comment by az0** [comment]
No, I am aware of BleachBit ever having any effect on printers. Could you elaborate more on the issue are you having with the printer? Did you use any  Winapp2.ini cleaners (which are enabled in the preferences)? Are you using Microsoft Windows?

**Comment by PolyEsterFhil** [comment]
Thank you for the response!!!  Fortunately, when I booted up today and ran a test page it printed it so it's back, for now.  It's been odd but this time I had to reinstall software and then generate a pin from the printer (HP) to get it to re-connect which I'd never seen before.
Yes, Windows 11 Home is my OS and the only time it seems to crash is when I run Bleach Bit but I love it and am truly grateful.  Not very tech savvy so I'm tech challenged...I appreciate everything you do!  Steve Mayberry 
    On Tuesday, October 15, 2024 at 12:46:23 AM EDT, Andrew Ziem ***@***.***> wrote:   

 


No, I am aware of BleachBit ever having any effect on printers. Could you elaborate more on the issue are you having with the printer? Did you use any Winapp2.ini cleaners (which are enabled in the preferences)? Are you using Microsoft Windows?

**Comment by az0** [comment]
I am closing because it seems to be resolved, but if this printing issue continues, reopen the same issue ticket here.

If it continues, it helps to narrow it down to a single cleaning option (meaning a single checkbox). When you enable them, read any warnings that pop up. If you clean with all checkboxes enabled, it's hard to know which option caused the problem, so to narrow it down, it helps to run cleaning in smaller groups or individually.
